### PR TITLE
Fix animation report

### PR DESCRIPTION
**Pull request by voger**
See #9061
I would like to hear comments and suggested improvements.

### CODE REVIEW

File: framework/source/class/qx/bom/element/AnimationHandle.js [hunk 1]
```diff
@@ -131,6 +132,11 @@ qx.Bootstrap.define("qx.bom.element.AnimationHandle",
         this.stopped = true;
         qx.bom.element.AnimationJs.stop(this);
       }
+    },
+
+    __setEnded : function(){
```
Comment (oetiker) [inline framework/source/class/qx/bom/element/AnimationHandle.js]: documentation is missing here

Comment (voger) [inline framework/source/class/qx/bom/element/AnimationHandle.js]: I though it was not needed since it was a private function. I will promptly fix it.

**Comment by johnspackman** [comment]
LGTM

**Comment by level420** [comment]
We have 4 thumbs up in the head description and the LGTM comment of @johnspackman. That makes 5 votes.
Merging.
Thank you @voger !

**Pull request merged**